src/briefcase/integrations/android_sdk.py

```diff
@@ -1274,6 +1263,12 @@ def logcat(self):
                 ],
                 env=self.android_sdk.env,
                 check=True,
+                stream_output=True,
             )
         except subprocess.CalledProcessError as e:
-            raise BriefcaseCommandError("Error starting ADB logcat.") from e
+            # If the user sends CTRL+C:
+            #  - on Windows, adb returns 0xC000013A (STATUS_CONTROL_C_EXIT)
+            #  - on Linux/macOS, adb returns -2.
+            # Exit normally since the user was instructed to use CTRL+C.
+            if e.returncode not in {-2, 0xC000013A}:
```

Comment:
In my testing, `adb` returned the unsigned decimal representation of `0xC000013A`: 3221225786. 

People have reported the signed representation, that is -1073741510, showing up in other applications.

I figured handling both is probably unnecessary at this point....especially if this isn't even reliable anyway....given I just found this through my own testing.